src/homepageExperience/containers/HomepagePythonWizard.tsx

```diff
@@ -55,6 +57,12 @@ export class HomepagePythonWizard extends PureComponent<null, State> {
       case 6: {
         return <ExecuteQuery />
       }
+      case 7: {
+        return <ExecuteAggregateQuery />
+      }
+      case 8: {
+        return <Finish />
+      }
```

Comment:
along with this, let's add a check for the `Next` button that if we're on step 8, don't render it. We should probably do the same thing with step 1 and the `Previous` button.

Reply:
Done!

Reply:
I used the `status` prop of out Button component.